specification/cognitiveservices/data-plane/TextAnalytics/readme.md

```diff
@@ -113,17 +113,17 @@ namespace ConsoleApp1
 
 > see https://aka.ms/autorest
 
-The current release is `release_2_0`.
+The current release is `release_2_1`.
```

Comment:
Why do you remove release_2_0 considering 2.1 is still in preview?

Reply:
I'm not sure what's the correct value here. Am I suppose to add the new version side-by-side or leave only the non-preview version?

Reply:
@assafi is the 2.1 API in "preview" state?

@jianghaolu if 2.1 is a preview version, are you suggesting to leave `release_2_0` in configuration and add the extra tag for `2.1`?